charts/velero/values.yaml

```diff
@@ -398,15 +402,15 @@ backupsEnabled: true
 # Whether to create volumesnapshotlocation crd, if false => disable snapshot feature
 snapshotsEnabled: true
 
-# Whether to deploy the restic daemonset.
-deployRestic: false
+# Whether to deploy the node-agent daemonset.
+defaultNodeAgent: false
```

Comment:
Have you considered using the name `deployNodeAgent`?

Reply:
yes, I've changed it to deployNodeAgent